engine/tests/phpunit/Elgg/TestCase.php

```diff
@@ -19,6 +20,16 @@
 	 */
 	public function __construct($name = NULL, array $data = array(), $dataName = '') {
 		self::bootstrap();
+
+		// force plugins->find() to return []
+		$plugins_mock = $this->getMockBuilder(Plugins::class)
```

Comment:
I refactored out the need for a mock (which required a test case instance). Separate SP for testing isn't worth trouble for now.